content/guides/figma/getting-started.mdx

```diff
@@ -54,3 +106,18 @@ To make components dynamic, we favor [component properties <LinkExternalIcon />]
 Whenever possible use components as they are without *detaching*.
 
 <a href="https://help.figma.com/hc/en-us/articles/360038662654-Guide-to-components-in-Figma">Learn more about Figma components <LinkExternalIcon /></a>
+
+## Using variable themes
+
+Primer Web provides *light mode* and *dark mode* using figma `variables`. This means you don't need a plugin to change between modes. Simply select any `frame` that uses `variables` or components with `variables`. Select the mode from the dropdown <svg class="svg" xmlns="http://www.w3.org/2000/svg" width="14" height="12" viewBox="0 0 14 12"><path fill="#000" fill-opacity="1" fill-rule="evenodd" d="m5 1.381-4 2.31v4.618l4 2.31 4-2.31V3.691L5 1.38zm5 1.732L5 .227 0 3.113v5.774l5 2.887 5-2.887V3.113zm3 5.196-5 2.887 1 .578 5-2.887V3.113L9 .227 8 .804l5 2.887v4.618zM6 6c0 .552-.448 1-1 1-.552 0-1-.448-1-1 0-.552.448-1 1-1 .552 0 1 .448 1 1z"></path></svg> in the *layer* section in the right sidebar
+
+<ImageBox caption="Switching the variable mode changes all nested items">
```

Comment:
I love little snippets. 😍

The GIF's quality isn't great, but it might be hard to fix unless we change it to a video. We should see what the team thinks, but I think we've talked about using videos instead of GIFs because they're easier for everyone to use.

Reply:
+1 for using a video instead of a GIF here

Reply:
How can I add an autoplaying video?

Reply:
There's a component called `CustomVideoPlayer`. It accepts all the same props as the HTML [<video>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video) tag, so you can pass `autoplay`.

Reply:
Great, thanks, used that now. It does not support `muted` though, right?